docs/commands/intend.md

```diff
@@ -28,29 +28,29 @@ The intend command will return the full request and response of the interaction
 
 By default, the system will:
 
-* Use the Intent Model and Sample Utterances in the speechAssets folder under the current working directory
+* Use the Interaction Model in the models folder under the current working directory
+* If there's no Interaction Model, use the Intent Model and Sample Utterances in the speechAssets folder under the current working directory
 * Use the service currently running via the `bst proxy` command
 
 If no service is currently running via bst proxy, and HTTP endpoint can be specified with the `--url` option:
 ```
 $ bst intend HelloIntent --url https://my.skill.com/skill/path
 ```
 
-## Speech Asset Format and Location
-If your speech assets  (Intent Model and Sample Utterances) are not stored under ./speechAssets, you can use an option to specify another location.
+## Interaction Model Format and Location
+If your Interaction Model is not stored under ./models, or you have multiple locales, you can use an option to specify another location.
 
 By default, we look for:
 
-* `./speechAssets/IntentSchema.json`
-* `./speechAssets/SampleUtterances.txt`
+* `./models/en-US.json`
 
 Example:
 ```
-$ bst intend HelloIntent -i interactions/IntentSchema.json -s interactions/SampleUtterances.txt
+$ bst intend HelloIntent -m model/en-UK.json
 ```
 
-The format of these files is the same as they are entered in the Alexa Skill configuration.  
+The format of these files is the same as they are entered in the Alexa Skill configuration.
```

Comment:
Do you mean via the online console? They are unfortunately not the same - not even when using the dialog builder.

I would combine this and the next sentence and say:
"These files are JSON, and typically defined by the ASK CLI tool from Amazon."